### PR TITLE
Last minor fixes (APPS-2054)

### PR DESCRIPTION
**Pull request by Fabtron**
* The local price has been updated to include a local price for deposit return voucher so that the total price is always up to date.
* The displayname for the deposit return voucher has been changed to return the total price instead of the scanned code.
* Also so some minor changes that are required.

APPS-2054

### Definition of Done

- [x] Issue is linked
- [ ] All requirements of the issue are fulfilled
- [ ] Changelog is updated
- [ ] Documentation is updated
- [ ] [Self-Review](https://www.nerdwallet.com/blog/engineering/why-you-should-be-doing-self-reviews/)
- [ ] Review with the Product Owner _(Release-Variante o. Minified Build)_

#### App Tests
- [ ] Minified build has been tested _(aka. Release Build)_
- [ ] Environments have been taken care of _(Production/Staging)_
- [ ] Supported languages have been tested
- [ ] Light-/Dark-Mode has been tested
- [ ] Edge-Cases have been tested
- [ ] Android API Levels have been taken care of _(minSdk?)_

#### Testing
- [ ] Tests have been written _(aka Unit-Tests, Integration-Tests, ...)_

### CODE REVIEW

**Review by cmaier — CHANGES_REQUESTED** [review]

File: core/src/main/java/io/snabble/sdk/shoppingcart/ShoppingCart.kt [hunk 1]
```diff
@@ -1199,20 +1199,26 @@ class ShoppingCart(
...
+                            cart?.project?.roundingMode,
+                            cart?.project?.customerCardId
+                        ) ?: 0
+                    }
+                    ItemType.DEPOSIT_RETURN_VOUCHER -> {
+                        return depositReturnVoucher?.lineItems?.sumOf { it.totalPrice } ?: 0
+                    }
+                    else -> {
+                        return 0
                     }
```
[the remark this answers is not in the document]
Comment (Fabtron) [reply core/src/main/java/io/snabble/sdk/shoppingcart/ShoppingCart.kt:1221]: changed
